src/components/Flag/Flag.styl

```diff
@@ -16,10 +16,13 @@
  */
 
 
-.f16, .f32 {
+.f32 {
     .flag {
         vertical-align: middle;
         margin-bottom: 0.1em;
     }
+    &.flag-small {
+      transform: scale(0.5, 0.5);
+      margin: -8px -8px -8px -8px;
```

Comment:
I don't love negative margins, but without this `f32` insisted on rendering a 32px container around the downscaled image - open to other ideas if you have them but this appears to work and maintain the layouts as they are now.

Reply:
Yeah it seemed a little weird at first glance, but I messed around a bit with some alternative approaches and they seemed to have various faults, and this seems to work well. Thanks for the solution @tylerbarker !

Tagging @benjaminpjones as he's stronger with some of this stuff than I for some additional sanity checking.

Reply:
Thanks for the followup @benjaminpjones !